model/clusters_mgmt/v1/cluster_resource.model

```diff
@@ -31,6 +31,8 @@ resource Cluster {
 		// If false it will only delete from OCM but not the actual cluster resources.
 		// false is only allowed for OCP clusters. true by default.
 		in Deprovision Boolean = true
+		// Dry run flag is used to check if the operation can be completed, but won't delete.
```

Comment:
NP: please add an empty line before this one.

Reply:
got it.